server/migrations/init.sql

```diff
@@ -21,4 +21,19 @@ INSERT INTO rentr_user(email, password) VALUES
 ('azizul@gmail.com', 'azizul123'),
 ('nathan@gmail.com', 'nathan123');
 
-SELECT * FROM rentr_user;
\ No newline at end of file
+SELECT * FROM rentr_user;
+
+CREATE TABLE IF NOT EXISTS rentr_listing (
+    id BIGSERIAL PRIMARY KEY, 
+    title VARCHAR(100) NOT NULL,
+    description VARCHAR(5000) NOT NULL,
+    num_bedroom VARCHAR(10) NOT NULL,
```

Comment:
Depending on whether we go with a dropdown list (which I think would be good), I think a foreign key to a bedroom dropdown options table could be a good idea. This helps to ensure that dropdown options are consistent throughout the application (i.e. search vs create a listing).

Reply:
will work on that in another issue or PR.